docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-setting-up-google-tag-manager.md

```diff
@@ -70,34 +70,46 @@ This event is commonly attached to the promotion banners carousel displayed by t
-| vtex:promotionClick | [promoClick](https://developers.google.com/tag-manager/enhanced-ecommerce#promo-clicks) | [select_promotion](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#select_promotion) |
-| vtex:productView | [detail](https://developers.google.com/tag-manager/enhanced-ecommerce#details) | [view_item](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#view_item) |
-| vtex:productImpression | [impressions](https://developers.google.com/tag-manager/enhanced-ecommerce#product-impressions) | [view_item_list](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#view_item_list) |
-| vtex:productClick | [click](https://developers.google.com/tag-manager/enhanced-ecommerce#product-clicks) | [select_item](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#select_item) |
-| vtex:addToCart | [add](https://developers.google.com/tag-manager/enhanced-ecommerce#add) | [add_to_cart](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#add_to_cart) |
-| vtex:removeFromCart | [remove](https://developers.google.com/tag-manager/enhanced-ecommerce#remove) | [remove_from_cart](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#remove_from_cart) |
-| vtex:cartLoaded| [checkout](https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce#checkout) | [begin_checkout](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtm#begin_checkout) |
-| vtex:orderPlaced | [purchase](https://developers.google.com/tag-manager/enhanced-ecommerce#purchases) | [purchase](https://developers.google.com/analytics/devguides/collection/ga4/reference/events#purchase) |
-
-
-### GA4 events
-
-Besides the events listed in the previous sections, GA4 has new events that stores can start tracking. The full list can be found on [Google documentation](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtm).
-
-The GTM app listens to the following events and sends them in the following corresponding GA4 format:
-
-| VTEX          | GA4   |
-| ------------- | ----- |
-| vtex:addPaymentInfo | add_payment_info |
-| vtex:addShippingInfo | add_shipping_info |
-| vtex:login | login |
-| vtex:signUp | sign_up |
-| vtex:viewCart | view_cart |
-| vtex:beginCheckout | begin_checkout |
-| vtex:refund | refund |
-| vtex:addToWishlist | add_to_wishlist |
-| vtex:search | search |
-| vtex:share| share |
+| VTEX                   | UA                                                                                              | GA4                                                                                                                                      |
```

Comment:
[Preview](https://github.com/filipewl/dev-portal-content/blob/fix/ga4-docs/docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-setting-up-google-tag-manager.md) output for these changes:

![CleanShot 2023-05-11 at 14 09 47@2x](https://github.com/vtexdocs/dev-portal-content/assets/381395/90bdf9a8-306e-4e93-ae95-159fb80fd88f)